token/token.go

```diff
@@ -51,8 +51,20 @@ const (
 	GeminiServer       = "fabric8-gemini-server"
 )
 
+var defaultManager Manager
+
+// DefaultManager creates the default manager if it has not created yet.
+// This function must be called in main to make sure the default manager is created during service startup.
+func DefaultManager(config Configuration) (Manager, error) {
+	var err error
+	if defaultManager == nil {
```

Comment:
Do you think it's worth putting some synchronization around this?

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/659/commits/8668cfa5b59bbce00e19baf6a0f85a02968e4294